### PR TITLE
Life cycle

### LINKED ISSUE #5350: Lifecycle improvements
Migration events create a new cohort from each cohort that it migrating.  Need to consolidate into a single new cohort otherwise we end up with to many.
Cohorts need to hold information about where they came from.

### PR DESCRIPTION
**Pull request by HamishBrownPFR**
resolves #5350

### CODE REVIEW

File: Models/Lifecycle/Cohort.cs [hunk 1]
```diff
@@ -26,7 +27,10 @@ public class Cohort
         public double Progeny { get; set; }
 
         /// <summary>Number of migrants leaving this cohort</summary>
-        public double Migrants { get; set; }
+        public double Emigrants { get; set; }
+
+        /// <summary>Information about where this cohort came from</summary>
+        public SourceInfo sorceInfo { get; set; }
```
Comment (hol353) [inline Models/Lifecycle/Cohort.cs:33]: Can this be spelt correctly? Will potentially save having to write a converter later.

Comment (hol430) [reply Models/Lifecycle/Cohort.cs:33]: Maybe it's supposed to be emigrants (electronic migrants) as in ebook?

Comment (HamishBrownPFR) [reply Models/Lifecycle/Cohort.cs:33]: It is spelt correctly.  Migration is the net term for moving, immigration is moving in and emigration is moving out.

Comment (zur003) [reply Models/Lifecycle/Cohort.cs:33]: Yes, but "sorceInfo" does seem like an odd spelling.

Comment (HamishBrownPFR) [reply Models/Lifecycle/Cohort.cs:33]: Information about the source of a cohort.  Suggestions?

Comment (hol353) [reply Models/Lifecycle/Cohort.cs:33]: Can we spell it sourceInfo rather than sorceInfo?

Comment (HamishBrownPFR) [reply Models/Lifecycle/Cohort.cs:33]: Ah, I see what you are on about now.  Spellings not my strong point.

**Comment by HamishBrownPFR** [comment]
@hol353 spelling fixed

**Pull request merged**